identity/validator_queue_test.go

```diff
@@ -0,0 +1,100 @@
+package identity
+
+import (
+	"os"
+	"testing"
+
+	"github.com/Oneledger/protocol/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+var vq1 = utils.NewQueued([]byte("a"), 200, 1)
+var vq2 = utils.NewQueued([]byte("b"), 1000, 2)
+var vq3 = utils.NewQueued([]byte("c"), 300, 3)
+var vq4 = utils.NewQueued([]byte("d"), 2000, 4)
+
+var Vq *ValidatorQueue
+
+func setup() {
+	logger.Debug("This is main setting up")
+}
+
+func shutdown() {
+	logger.Debug("This is main tearing down")
+}
+
+func TestMain(m *testing.M) {
+	setup()
+	code := m.Run()
+	shutdown()
+	os.Exit(code)
+}
+
+func subtestsetup(funName string) {
+	logger.Debug("This is subtest setting up for", funName)
```

Comment:
Nothing gets logged to test, when running `go test` the output will be suppressed